<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROOTDraw</class>
 <widget class="QMainWindow" name="ROOTDraw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>328</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Choose Draw File</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="2">
      <widget class="QRadioButton" name="btnHGDraw">
       <property name="text">
        <string>High Gain</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="2">
      <widget class="QPushButton" name="btnFileClose_2">
       <property name="text">
        <string>Close Save File</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QRadioButton" name="btnTDCDraw">
       <property name="text">
        <string>TDC</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>Select Channel:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QSpinBox" name="boxFiberCh_2">
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="lblReadFileName">
       <property name="text">
        <string>Data*.root</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="5">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnFileChoose">
       <property name="text">
        <string>Choose File</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QPushButton" name="btnFileDraw">
       <property name="text">
        <string>Draw  File</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QRadioButton" name="btnLGDraw">
       <property name="text">
        <string>Low Gain</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Select Queue:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="frmDraw" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>1101</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpPeakStat">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>741</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Peak Stat</string>
    </property>
    <widget class="QTableWidget" name="tablePeaks">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>26</y>
       <width>511</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>191</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="0" colspan="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Rebin:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineSigmaInput">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Peak Find Args:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ADC/PE</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Gain:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineGainOutput">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnFitPeaks">
        <property name="text">
         <string>Fit Peaks</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Sigma:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="boxRebin">
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>SPE Peak:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Ped Peak:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="lineSPE"/>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="linePed"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
